migrate.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
```

Comment:
agree with any ways we can make the thing more picky. Its only gonna run once, but i want to make sure it does the right thing.